### ISSUE TITLE
Not easy to share FAQ question

### ISSUE THREAD

**Issue by thanhson1085** [tomochain/docs#139]
I agree that use "quote" for the question make FAQ quite nice.
But "quote" is not SEO friendly and not easy to share the question for other people.

Use "header" for the question is make the page more SEO friendly and easy to share.
e.g:

Current:
```
!!! quote "What is TomoChain?"
    TomoChain is an innovative solution to the scalability problem of the Ethereum blockchain, and other current blockchain platforms.
	TomoChain relies on a system of 150 Masternodes with Proof of Stake Voting (POSV) consensus that can support low transaction fees and 2-second transaction confirmation times.
	Security, stability and chain finality are guaranteed via novel techniques such as double validation, staking via smart-contracts and true randomization processes.

	TomoChain supports all EVM-compatible smart-contracts, protocols, and atomic cross-chain token transfers.
	New scaling techniques such as sharding, EVM parallelisation, private-chain generation, hardware integration will be continuously researched and incorporated into TomoChain's Masternode architecture.
	It will be an ideal scalable smart-contract public blockchain for decentralized apps, token issuances and token integrations for small and big businesses.
```

should change to:
```
### What is TomoChain?
TomoChain is an innovative solution to the scalability problem of the Ethereum blockchain, and other current blockchain platforms.
TomoChain relies on a system of 150 Masternodes with Proof of Stake Voting (POSV) consensus that can support low transaction fees and 2-second transaction confirmation times.
Security, stability and chain finality are guaranteed via novel techniques such as double validation, staking via smart-contracts and true randomization processes.

TomoChain supports all EVM-compatible smart-contracts, protocols, and atomic cross-chain token transfers.
New scaling techniques such as sharding, EVM parallelisation, private-chain generation, hardware integration will be continuously researched and incorporated into TomoChain's Masternode architecture.
It will be an ideal scalable smart-contract public blockchain for decentralized apps, token issuances and token integrations for small and big businesses.
```

So if i want to share the question to people, I can use this link: `https://docs.tomochain.com/general/faq/#what-is-tomochain`

**Comment by damvananh91** [comment]
Agree for SEO-friendly.

**Comment by phamvancam2104** [comment]
Agree for SEO friendly

**Comment by DuqueKarl** [comment]
https://github.com/tomochain/docs/pull/143

**Comment by phamvancam2104** [comment]
closed in #143
